app/app.py

```diff
@@ -173,22 +173,32 @@ def _handle_premis_event(event: PremisEvent, mh_client: MediaHaven):
 
         s3_bucket = fragment_info["s3_bucket"]
         s3_object_key = fragment_info["s3_object_key"]
+        # If we have a collateral (subtitle): no need for een archivedEvent
+        if s3_bucket == 'mam-collaterals':
```

Comment:
'mam-collaterals' => "mam-collaterals"

Reply:
Since it's the default of black (https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings) most strings in our codebases use double quotes.